webapp/pages/profile/_id/_slug.spec.js

```diff
@@ -200,7 +162,10 @@ describe('ProfileSlug', () => {
           describe('pagination returned as many posts as available', () => {
             beforeEach(() => {
               const posts = [1, 2, 3, 4, 5, 6].map(id => {
-                return { ...aPost, id }
+                return {
+                  ...aPost,
+                  id,
+                }
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-14T10:59:37Z" title="Friday, June 14th 2019, 12:59:37 pm +02:00">Jun 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -200,7 +162,10 @@ describe('ProfileSlug', () => {
           describe('pagination returned as many posts as available', () => {
             beforeEach(() => {
               const posts = [1, 2, 3, 4, 5, 6].map(id => {
-                return { ...aPost, id }
+                return {
+                  ...aPost,
+                  id,
+                }
```


Not a fan of this formatting... wonder if this was `yarn lint --fix`??

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-06-14T11:14:18Z" title="Friday, June 14th 2019, 1:14:18 pm +02:00">Jun 14, 2019</time>_

---


It’s not my code. @mattwr18 
Probably @roschaefer wrote this. It should been overtaken from the master.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-14T13:59:17Z" title="Friday, June 14th 2019, 3:59:17 pm +02:00">Jun 14, 2019</time>_

---


I was referring to the formatting of the code... which it's seems this `PR` introduced, but I don't think it's a big deal...